src/DynamoApplications/PathResolvers.cs

```diff
@@ -33,7 +33,7 @@ public SandboxPathResolver(string preloaderLocation)
                 "DSOffice.dll",
                 "DSIronPython.dll",
                 "FunctionObject.ds",
-                "BuiltIns.ds",
+                "BuiltIn.ds",
```

Comment:
When the file name is changed, there are some other parts which need to be changed as well. E.g., refer the changes done in https://github.com/DynamoDS/DynamoRevit/pull/1665